src/common/components/card/Card.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { FunctionComponent } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import angleDownIcon from '../../../assets/icons/svg/angleDown.svg';
+import styles from './card.module.scss';
+import Button from '../button/Button';
+import Icon from '../icon/Icon';
+
+interface CardProps {
+  action: () => void;
+  actionText: string;
+  image: string;
+  primaryAction?: () => void;
+  primaryActionText?: string;
+  title: string;
+}
+
+const Card: FunctionComponent<CardProps> = ({
+  action,
+  actionText,
+  children,
+  image,
+  primaryAction,
+  primaryActionText,
+  title,
+}) => {
+  const { t } = useTranslation();
+
+  return (
+    <div className={styles.wrapper}>
+      <div className={styles.start}>
+        {/* TODO: alt, size */}
+        <img src={image} alt={'alt'} width="200" height="200" />
```

Comment:
For mobile I don't think you need to define the width, you can just let it fill the screen? 

And if you don't define height either you can allow the editors to choose the image aspect ratio (not sure if that is a good idea)